test/nodes/test_prompt_node.py

```diff
@@ -926,3 +926,27 @@ def test_simple_pipeline_batch_query_multiple_doc_list(self, prompt_model):
 def test_HFLocalInvocationLayer_supports():
     assert HFLocalInvocationLayer.supports("philschmid/flan-t5-base-samsum")
     assert HFLocalInvocationLayer.supports("bigscience/T0_3B")
+
+
+@pytest.mark.integration
+def test_chatgpt_direct_prompting(chatgpt_prompt_model):
+    skip_test_for_invalid_key(chatgpt_prompt_model)
+    pn = PromptNode(chatgpt_prompt_model)
+    result = pn("Hey, I need some Python help. When should I use list comprehension?")
+    assert len(result) == 1 and all(w in result[0] for w in ["comprehension", "list"])
+
+
+@pytest.mark.integration
```

Comment:
@anakin87 let's add one more unit test of using PromptNode("gpt-3.5-turbo") directly. Mostly so that @agnieszka-m can get a sense of what needs to be done to enable the usage of `gpt-3.5-turbo` model.

Reply:
test added